framework/index.scss

```diff
@@ -59,3 +59,4 @@
 @import 'atoms/images/images';
 @import 'atoms/icons/icons';
 @import 'atoms/buttons/buttons';
+@import 'atoms/horizontal-line/horizontal-lines';
```

Comment:
There's a "s" missing in the folder name! Should be `atoms/horizontal-lines/horizontal-lines`